src/app/views/merch/merch.component.html

```diff
@@ -0,0 +1,140 @@
+    </div>
+    <div>
+      <iframe id="no-target" name="no-target" style="display: none"></iframe>
+      <form
+        #merchForm
+        [formGroup]="form"
+        class="row csc-contact-form"
+        [action]="googleForm.url"
+        target="no-target"
+      >
+        <csc-input-container
+          class="col-6"
+          label="Brock Student Email"
+          labelLocation="top"
+        >
+          <input
+            [name]="googleForm.ids.email"
+            formControlName="email"
+            class="csc-input"
+            type="text"
+            placeholder="aa11bb@brocku.ca"
+          />
+          <span
+            class="signup-error"
+            *ngIf="
+              !form.controls.email.valid &&
+              (form.controls.email.touched || submitted)
+            "
+            >Please enter your Brock Student Email</span
+          >
+        </csc-input-container>
+        <csc-input-container
+          class="col-6"
+          label="Brock Student Id"
+          labelLocation="top"
+        >
+          <input
+            [name]="googleForm.ids.id"
+            formControlName="id"
+            class="csc-input"
+            type="text"
+            placeholder="7665540"
+          />
+          <span
+            class="signup-error"
+            *ngIf="
+              !form.controls.id.valid && (form.controls.id.touched || submitted)
+            "
+            >Please enter your Brock Student ID</span
+          >
+        </csc-input-container>
+        <csc-input-container class="col-6" label="Color" labelLocation="top">
+          <select
+            [name]="googleForm.ids.color"
+            formControlName="color"
+            (change)="changeColor($event.target.value)"
+          >
+            <option value="Black">Black</option>
+            <option value="White">White</option>
+          </select>
+          <span
+            class="signup-error"
+            *ngIf="
+              !form.controls.color.valid &&
+              (form.controls.color.touched || submitted)
+            "
+            >Please pick a color</span
+          >
+        </csc-input-container>
+        <csc-input-container class="col-6" label="Size" labelLocation="top">
+          <select [name]="googleForm.ids.size" formControlName="size">
+            <option value="S">S</option>
+            <option value="M">M</option>
+            <option value="L">L</option>
+            <option value="XL">XL</option>
+          </select>
+          <span
+            class="signup-error"
+            *ngIf="
+              !form.controls.size.valid &&
+              (form.controls.size.touched || submitted)
+            "
+            >Please pick a size</span
+          >
+        </csc-input-container>
+        Interac e-transfer <b>45 CAD</b> to <b>payments@brockcsc.ca</b>.<br />
+        Locate the refence number on the application, or in the notification
+        email.
```

Comment:
```suggestion
        Interac e-transfer <b>45 CAD</b> to <b>payments@brockcsc.ca</b>.<br />
        Locate the reference number on your banking app, or in the notification
        email.
```

May want to disambiguate what application is being referred to.